include/JobQueue.hh

```diff
@@ -78,6 +78,11 @@ void runJobs(JSContext *cx) override;
  */
 bool empty() const override;
 
+/**
+ * @return true if the job queue stops draining, which results in `empty()` being false after `runJobs()`.
```

Comment:
```suggestion
 * @return true if the job queue stopped draining, which results in `empty()` being false after `runJobs()`.
```

Reply:
I copied this from https://hg.mozilla.org/mozilla-central/file/d25c9bd/js/public/Promise.h#l84.

Reply:
I couldn't understand why you'll need to change this. 
Is this grammatically incorrect?

Reply:
indeed, yes, "stops" refers to the present or future, whereas "stopped" refers to the past